cfn.template

```diff
@@ -147,39 +147,51 @@ Resources:
                     regions=$(aws ec2 describe-regions --output text --query 'Regions[].RegionName')
                   fi
 
+                - # Deal with either "template" or "templates"
```

Comment:
Open to opinion on this but I thought it would be better if we allow the config files to say `templates`. This shim right here allows just `template` to work too.

Reply:
for backwards compatibility I think we should support both